### ISSUE TITLE
Reactions to Control Apollo

### ISSUE THREAD

**Issue by Radiumio** [CreepPork/Apollo#151]
Using Discord reactions, Apollo could generate a message (similar to how it does with server status), and put an emoji of the users' choice on the message. The message could contain basic stats of the server, maybe player count, map, etc, and whether or not maintenance mode is on. Upon clicking the reaction, the bot checks if you have the correct role to manage the bot, and if you do, it'll enable maintenance mode.

So basically, just a reaction message that allows you to toggle maintenance mode without flooding the channel with bot commands all the time. Could branch off into other features as well.
